packages/ocaml-android32.4.02.3/opam

```diff
@@ -3,7 +3,11 @@ maintainer: "whitequark@whitequark.org"
 patches: ["ar-fix.patch" "patches/ocamldoc.patch" "patches/ocamlmklib.patch"]
 substs: ["android.conf" "config/Makefile"]
 build: [
-  [make "world" "opt" "install"]
+  ["./configure"]
+  [make "world" "bootstrap" "opt" "opt.opt"]
+  [sudo umask 022]
+  [sudo make "install"]
+  [make "clean"]
```

Comment:
The changes in this file are not only unnecessary, but they will also leave you without a cross-compiler (instead, a regular compiler will be built) and with a broken system (because you are not supposed to install a cross-compiler as your global ocaml compiler).